devtools/test_dashboard/server.js

```diff
@@ -35,11 +35,7 @@ devtoolsConfig.entry = path.join(devtoolsPath, 'devtools.js');
 devtoolsConfig.output = {
     path: config.output.path,
     filename: 'test_dashboard-bundle.js',
-    library: {
-        name: 'Tabs',
-        type: 'umd',
-        umdNamedDefine: true
-    }
+    library: config.output.library
```

Comment:
Before this change we had `name: 'Tabs'`, don't we want to keep that?

Reply:
Hmm... If I want to maintain `Tabs` name, then there won't be a `Plotly` object attached to the `window`. 
What's your suggestion?

Reply:
Currently in the test dashboard we have BOTH `Plotly` (plotly.js) and `Tabs` (name is a historical artifact, but it's tools that I think might still be used in setting up the dashboard?)

Can't we just do something like:
```suggestion
    library: {...config.output.library, name: 'Tabs'}
```